src/common/options/mds.yaml.in

```diff
@@ -92,6 +92,24 @@ options:
   - mds
   flags:
   - runtime
+- name: mds_cache_quiesce_auth_recall
```

Comment:
I don't think this one is used anywhere, is it?

Reply:
look again